nautobot/extras/templatetags/plugins.py

```diff
@@ -55,6 +60,23 @@ def _get_registered_content(obj, method, template_context, return_html=True):
     return mark_safe(html)  # noqa: S308  # suspicious-mark-safe-usage -- we have to trust plugins to provide safe HTML
 
 
+def _get_registered_ui_content(obj, attr) -> list:
+    """Given an object and TemplateExtension attribute name, return all relevant extensions' content.
```

Comment:
Should we give an example on how to use this method? I know it is not user-facing.